reana_server/rest/workspaces.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Reana-Server Workspaces Endpoint."""
```

Comment:
Minor: for consistency I guess we should call it `Reana-Server Workspaces functionality Flask-Blueprint.` just like in other blueprints